static/app/views/replays/detail/network/index.tsx

```diff
@@ -348,55 +467,42 @@ const UnstyledHeaderButton = styled(UnstyledButton)`
   align-items: center;
 `;
 
-const StyledPanelTable = styled(PanelTable)<{columns: number}>`
-  grid-template-columns: max-content minmax(200px, 1fr) repeat(4, max-content);
-  grid-template-rows: 24px repeat(auto-fit, 28px);
-  font-size: ${p => p.theme.fontSizeSmall};
-  margin-bottom: 0;
+const NetworkTable = styled('div')`
+  list-style: none;
+  position: relative;
   height: 100%;
-  overflow: auto;
-
-  > * {
-    border-right: 1px solid ${p => p.theme.innerBorder};
-    border-bottom: 1px solid ${p => p.theme.innerBorder};
+  overflow: hidden;
+  border: 1px solid ${p => p.theme.border};
+  border-radius: ${p => p.theme.borderRadius};
+  padding-left: 0;
+  margin-bottom: 0;
+`;
 
-    /* Last column */
-    &:nth-child(${p => p.columns}n) {
-      border-right: 0;
-      text-align: right;
-      justify-content: end;
-    }
+const NetworkTableHeader = styled('div')<{columns: number}>`
+  color: ${p => p.theme.subText};
+  font-size: ${p => p.theme.fontSizeSmall};
+  font-weight: 600;
+  background: ${p => p.theme.backgroundSecondary};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
 
-    /* 3rd and 2nd last column */
-    &:nth-child(${p => p.columns}n - 1),
-    &:nth-child(${p => p.columns}n - 2) {
-      text-align: right;
-      justify-content: end;
-    }
-  }
+  max-height: ${headerRowHeight}px;
+  border-radius: 0;
+  line-height: 16px;
+  text-transform: none;
 
-  ${PanelTableHeader} {
-    min-height: 24px;
-    border-radius: 0;
-    color: ${p => p.theme.subText};
-    line-height: 16px;
-    text-transform: none;
-
-    /* Last, 2nd and 3rd last header columns. As these are flex direction columns we have to treat them separately */
-    &:nth-child(${p => p.columns}n),
-    &:nth-child(${p => p.columns}n - 1),
-    &:nth-child(${p => p.columns}n - 2) {
-      justify-content: center;
-      align-items: flex-start;
-      text-align: start;
-    }
-  }
+  border-right: 1px solid ${p => p.theme.innerBorder};
+  border-bottom: 1px solid ${p => p.theme.innerBorder};
 `;
 
 const SortItem = styled('span')`
   padding: ${space(0.5)} ${space(1.5)};
```

Comment:
Can we move this padding up into the `UnstyledHeaderButton` so click-targets a larger, and a little tweak like `${space(0.5)} ${space(1)} ${space(0.5)} ${space(1.5)};` so there's not too much space next to the arrow on the right.

Also, we should be able to move everything from `NetworkTableHeader` into here, and remove NetworkTableHeader